content/en/users/security/secrets-store/gui/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Secrets Store Web Interface
+linkTitle: Web Interface
+type: docs
+weight: 10
+description: >
+  The web interface of EGI Secrets Store
+---
+
+Although the recommended way to access and use EGI Secrets Store is via the
+[command-line interface](../cli), the service can also be accessed via its
+web interface.
+
+{{% alert title="Tip" color="info" %}} Tutorials about using the web interface
+are available on the
+[Hashicorp Vault site](https://developer.hashicorp.com/vault/tutorials/getting-started-ui).
+{{% /alert %}}
+
+## Using the web interface
+
+Follow the steps below to access the web interface of EGI Secrets Store:
+
+<!-- markdownlint-disable no-bare-urls -->
+- Open https://vault.services.fedcloud.eu:8200 in your browser.
+- Choose the _OIDC authentication_ method in the pulldown menu, then click
+_Sign in with OIDC provider_.
+- Login via _EGI Check-in_ and authorize the Vault GUI.
+<!-- markdownlint-enable no-bare-urls -->
+
+![Sign-in](vault-login-oidc.png)
+
+The main page will open, with a top folder **secrets** that contains a space for
+each user.
+
+![Main page](vault-main-window.png)
+
+### Browse secrets
```

Comment:
```suggestion
### Browsing secrets
```

Reply:
Disagree. This is at a different level than the previous one with which you tried to uniformize. Level 2 titles are using gerund (as alternative, or short for of "how to..."). Level 3 titles are commands (also in the CLI page), thus nouns (one time action).
https://writingcenter.gmu.edu/writing-resources/grammar-style/choosing-between-infinitive-and-gerund-to-do-or-doing